docs/adr/0004-adr-db-seeding.md

```diff
@@ -0,0 +1,66 @@
+# Database seeding paradigm
+
+[EASI - 2156](https://jiraent.cms.gov/browse/EASI-2156)
+
+MINT has a need to seed data in the database to rapidly develop and test the application. EASi has historically used a Go file to write directly to the database layer to seed data. This required a good deal of maintenance and manual coding.
+
+As an alternative, we have been using a postgres db dump file to quickly seed the data. While simpler, this also introduces some complexity updating the seed file when the database structure is modified in any way.
+
+## Considered Alternatives
+
+* Direct interaction with the database store in Go
+* Continue using db dump file, but let migrations update the dump file
+* Continue using db dump file, but write a Go program to update the seed file programmatically
+* Implement a Go seed program that utilizes the schema resolver methods
+
+## Decision Outcome
+
+* Chosen Alternative: **Implement a Go seed program that utilizes the schema resolver methods**
```

Comment:
Can you link the ticket that was created to implement this in this PR? Just in the comments is fine (or in this section of the ADR, probably wouldn't hurt for visibility!)

Reply:
I started adding in the main message yesterday, but I definitely didn't add any description haha, that would be helpful. I think you are right, it would be clear to have it in the actual ADR as well, so I will add there too.

The future ticket is [EASI-2187](https://jiraent.cms.gov/browse/EASI-2187)